Sources/PointFree/Subscribe.swift

```diff
@@ -6,19 +6,35 @@ import Prelude
 import Tuple
 
 public struct SubscribeData: Codable {
-  public let pricing: Pricing
-  public let token: Stripe.Token.Id
-  public let vatNumber: String
+  public typealias Coupon = Tagged<SubscribeData, String>
+
+  public private(set) var coupon: Coupon?
```

Comment:
this is technically the coupon code... do you think I should rename?

Reply:
I don't mind either way. This is consistent with Stripe at least.